docs/Installation_Alternative.rst

```diff
@@ -36,13 +36,13 @@ you can use one of the following simple commands:
 
 -  using ``easy_install`` (old tool, but still works)::
 
-       easy_install --prefix /tmp easybuild
+       easy_install --prefix /tmp/$USER easybuild
```

Comment:
I really don't like `/tmp` for such things. That's only for temporarily files that should not survive reboot. `/var/tmp` might be a slightly better location.
But I would really for go something under `$HOME` ? I've been using `$HOME/opt` since forever without issue? If we use `--prefix`, I don't really see a lot of problems?